operations/helm/charts/mimir-distributed/templates/gossip-ring/gossip-ring-svc.yaml

```
@@ -5,6 +5,8 @@ metadata:
   labels:
     {{- include "mimir.labels" (dict "ctx" . "component" "gossip-ring") | nindent 4 }}
   namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    {{- toYaml .Values.gossipRing.service.annotations | nindent 4 }}
```

We should change this also

```suggestion
    {{- toYaml .Values.gossip_ring.service.annotations | nindent 4 }}
```